ingeniamotion/capture.py

```diff
@@ -151,6 +152,7 @@ def create_empty_monitoring(self, servo: str = DEFAULT_SERVO) -> Monitoring:
             NotImplementedError: If an wrong monitoring version is requested.
 
         """
+        self.mc._get_drive(servo)
```

Comment:
You added a `self.mc._get_drive(servo)`, but you don't need the output, which is weird. In this case, I think that `self._check_version(servo)` should crash when trying to get the drive, I don't think it's necessary to force a `self.mc._get_drive(servo)` call.

Reply:
Ok, I see it.